frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -33,6 +34,12 @@ export default class PublicGroupHero extends Component {
   render() {
     const { group, i18n, session, hasHost, canEditGroup, groupForm, appendEditGroupForm} = this.props;
 
+    const titles = Object.keys(processMarkdown(group.longDescription));
+
+    const getAnchor = (title) => {
+      return title.toLowerCase().replace(' ','-').replace(/[^a-z0-9\-]/gi,'')
```

Comment:
The `toLowerCase()` call removes the need to make the regex case insentive (you can remove the `i`)